optimize/client/docker-compose.yml

```diff
@@ -70,7 +70,7 @@ services:
       - keycloak
     restart: on-failure
     container_name: identity
-    image: camunda/identity:latest
+    image: camunda/identity:${IDENTITY_VERSION:-8.5.0}
```

Comment:
should we default this to 8.5.0? this will kinda require us to update this with each new release

Reply:
I did like that to be consistent with how we currently do it for other images.
using `latest` is also problematic and can cause problems, so i would prefer to have a specific version.